solutions/src/main/java/ru/sberbank/school/task02/ServiceFactoryImpl.java

```diff
@@ -0,0 +1,21 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+
+public class ServiceFactoryImpl implements ServiceFactory {
+
+
+
+    @Override
+    public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) throws FxConversionException {
+
+        if (externalQuotesService == null) {
+            throw new FxConversionException("Неправильная ссылка на объект котировок ExternalQuotesService");
+        }
```

Comment:
&#39;}&#39; at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a562563187>*